src/utils/pick-profile.ts

```diff
@@ -19,4 +20,19 @@ const pickProfile = async (): Promise<string | undefined> => {
   return await vscode.window.showQuickPick(profileNames);
 };
 
+export const getProfiles = () => {
+  const credentialPath = `${os.homedir()}/.newrelic/credentials.json`;
+  const defaultPath = `${os.homedir()}/.newrelic/default-profile.json`;
+
+  const profiles = JSON.parse(fs.readFileSync(credentialPath));
+  const currentDefault = JSON.parse(fs.readFileSync(defaultPath));
```

Comment:
We could add error handling around the reading of the files and letting the user know that the they need to add a profile (or select a default) instead of them left wondering what to do if they see an error about a missing credentials file. I'm not sure that that file will exist on initial install of the CLI.